profile/README.md

```diff
@@ -0,0 +1,19 @@
+# Hello! 👋
+
+![Never become yet another.](./banner.png)
+
+This is the GitHub organization for [UpCloud](https://upcloud.com/), a leading European cloud service provider with a global cloud platform spanning four continents. UpCloud offers Cloud Servers, Managed Kubernetes, Managed Databases, Object Storage and more.
```

Comment:
Oh yeah are we still doing the _"we will use British English in our marketing"_ thing? If so, then "organisation" with an "s" instead of "z".

Reply:
I'll update that 👌